ClosedXML_Tests/Utils/StreamHelper.cs

```diff
@@ -156,9 +157,41 @@ private static string RemoveIgnoredParts(this string s, Uri uri, Boolean ignoreC
             if (ignoreGuids)
                 s = RemoveGuids(s);
 
+            s = RemoveNonCodingXmlFormatDiff(s);
+
             return s;
         }
 
+        private static string RemoveNonCodingXmlFormatDiff(string s)
+        {
+            try
+            {
+                var original = XDocument.Parse(s);
+
+                var normalized = new XDocument(original);
+
+                string v = normalized.ToString();
+                return v;
+            }
+            catch (System.Xml.XmlException ex)
+            {
+                if (ex.Message.Contains("Data at the root level is invalid."))
```

Comment:
is this a good idea considering translations? I suppose System.Xml is translated.
The normal way to check for such problems is to iterate using an XmlReader

Reply:
well.... its test code. i wont put any effort into this.